src/parser/jobs/whm/modules/Lilies.tsx

```diff
@@ -0,0 +1,167 @@
+		maximum: this.parser.patch.before('6.1') ? LILY_60_TIME_REQUIRED : LILY_TIME_REQUIRED,
+		onExpiration: this.onLilyGeneration.bind(this),
+		graph: {
+			label: <Trans id="whm.gauge.resource.lily-timer">Lily Timer</Trans>,
+			color: LILY_COLOR.fade(TIMER_FADE),
+		},
+
+	}))
+
+	private bloodLilyGauge = this.add(new CounterGauge({
+		maximum: BLOOD_LILY_BLOOM,
+		initialValue: 0,
+		graph: {
+			label: <Trans id="whm.gauge.resource.bloodlily">Blood Lily</Trans>,
+			color: BLOODLILY_COLOR.fade(BLOODLILY_FADE),
+		},
+	}))
+
+	override initialise() {
+		super.initialise()
+
+		const playerFilter = filter<Event>().source(this.parser.actor.id)
+		this.addEventHook(playerFilter.type('action').action(oneOf(this.lilyConsumers)), this.onLilySpend)
+		this.addEventHook(playerFilter.type('action').action(oneOf(this.bloodLilyConsumers)), () => this.bloodLilyGauge.spend(MISERY_COST))
+		this.addEventHook('complete', this.onComplete)
+		this.lilyTimer.start()
+	}
+
+	private onLilyGeneration() {
+		this.lilyGauge.generate(1)
+		if (!this.lilyGauge.capped) {
+			this.lilyTimer.start()
+		}
+	}
+
+	private onLilySpend() {
+		this.bloodLilyGauge.capped ? this.bloodLilyGauge.overCap += 1 : this.bloodLilyGauge.modify(1)
+		if (this.lilyTimer.expired) {
+			this.lilyTimer.start()
+		}
+		this.lilyGauge.spend(1)
+
+	}
+
+	private calculateLostLilies(): number {
+		const lostLilies = Math.floor(this.lilyTimer.getExpirationTime() / LILY_TIME_REQUIRED)
+		return lostLilies
+	}
+
+	private onComplete() {
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.AFFLATUS_MISERY.icon,
+			content: <Trans id="whm.lily-blood.suggestion.content">
+				Use <DataLink action="AFFLATUS_MISERY" /> to avoid wasting Blood lily growth.
+			</Trans>,
+			tiers: {
+				1: SEVERITY.MINOR,
+				2: SEVERITY.MEDIUM,
+				3: SEVERITY.MAJOR,
+			},
+			value: this.bloodLilyGauge.overCap,
+			why: <Trans id="whm.lily-blood.suggestion.why">
+				<Plural value={this.bloodLilyGauge.overCap} one="# Blood lily" other="# Blood lilies" /> did not bloom due to early lily use.
+			</Trans>,
+		})
+		)
+
+		if (this.calculateLostLilies() > 0) {
+			let content
+			let tiers
+
+			if (this.parser.patch.before('6.1')) {
+				content = <Trans id="whm.lily-cap.suggestion.6_0_content">
+					Use <DataLink action="AFFLATUS_RAPTURE" /> or <DataLink action="AFFLATUS_SOLACE" /> before using other GCD heals. It's okay to cap your lilies if you don't need to heal, move, or weave with them.
+				</Trans>
+
+				tiers = {
+					1: SEVERITY.MINOR,
+				}
+			} else {
+				content = <Trans id="whm.lily-cap.suggestion.6_1_content">
+					Use <DataLink action="AFFLATUS_RAPTURE" /> or <DataLink action="AFFLATUS_SOLACE" /> before using other GCD heals. It's okay to overheal with your lilies as they are to be used for mana management and movement aswell as healing.
+				</Trans>
+
+				tiers = {
+					1: SEVERITY.MEDIUM,
+					3: SEVERITY.MAJOR,
+				}
+			}
```

Comment:
Assign the strings to consts, then just swap out the const used. Tiers are also best done in consts for easier maintenance, for all suggestions in this file.